exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,14 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+
+  return first = arr[0];
```

Comment:
You don't return the result of assigning a variable (`undefined`), just the value (in this case `return arr[0]`)